lib/clueNumbersFromBoard.ts

```diff
@@ -1,72 +1,105 @@
-import { getTile } from "./getTile"
-import type { CrosswordJSON, Position } from "./types"
+import type { CrosswordJSON, Tile, Position } from "./types"
 
-export const getCluePositionsForBoard = (tiles: CrosswordJSON["tiles"]): Position[] => {
-  const spots = _getPossibleTiles(tiles)
-  const across = _acrossCluePositions(spots)
-  const down = _downCluePositions(spots)
-  return _getOrderedPositionsForSpots(spots, across, down)
+type PositionWithTiles = {
+  position: Position
+  tiles: {
```

Comment:
This is the main change here: we now collect all tiles associated with a given clue so we can detect rebuses and know the true length of a clue on the grid.